src/components/Upload/Forms/v1StyleFormValidator.test.js

```diff
@@ -2,93 +2,90 @@ import { validate, validateNotEmpty, validateDate } from './v1StyleFormValidator
 import React from 'react';
 
 describe("validate", () => {
-	it("should return 'Required' in errors.submitterFirstName when no first name supplied", () => {
-		let values = { "submitterFirstName": '' };
-		let result = validate(values);
-		expect(result.submitterFirstName).toEqual('Required');
-	});
-	it("should return undefined in errors.submitterFirstName when a first name supplied", () => {
-		let values = { "submitterFirstName": 'firstName' };
-		let result = validate(values);
-		expect(false).toEqual(result.hasOwnProperty('submitterFirstName'));
-	});
-	it("should return 'Required' in errors.submitterLastName when no last name supplied", () => {
-		let values = { 'submitterLastName': '' };
+
+	let values = {
+		submitter: {
+			id: undefined
+		}
+	};
+
+	it("should return 'Required' in errors.submitter.id when no id supplied", () => {
+		values.submitter.id = "";
 		let result = validate(values);
-		expect(result.submitterLastName).toEqual('Required');
+		expect(result.submitterId).toEqual('Required');
 	});
-	it("should return undefined in errors.submitterLastName when a last name supplied", () => {
-		let values = { 'submitterLastName': 'last' };
+	it("should return undefined in errors.submitter.id when id supplied", () => {
+		values.submitter.id = "1234";
 		let result = validate(values);
-		expect(false).toEqual(result.hasOwnProperty('submitterLastName'));
+		expect(false).toEqual(result.hasOwnProperty('submitterId'));
 	});
 	it("should return 'Required' in errors.institution when no institution supplied", () => {
-		let values = { "institution": '' };
+		values.institution = "";
 		let result = validate(values);
 		expect(result.institution).toEqual('Required');
 	});
 	it("should return undefined in errors.institution when a institution supplied", () => {
-		let values = { "institution": 'name' };
+		values = {...values, "institution": 'name' };
```

Comment:
Why the change to pass in the values with a submitter?

Reply:
Because otherwise it was throwing an error for each test that the submitter was undefined. There's probably a better way to do it but it's what I came up with.